department-of-veterans-affairs/vets-website
# 72960 landing page first draft

## Summary

- All changes are within src/applications/representatives. The primary change to this directory is that the child of the App is now the Landing Page (a new page styled to mirror the VA.gov landing page with a slight different look-and-feel styling) which has a Link to the Dashboard Page
- The Sign in modal will be added in a subsequent PR and the RequiredLoginView will be uncommented to ensure the Dashboard cannot be viewed until a user is logged in
- This is not a bug fix
- Added a rough draft of Landing Page to bring the user flow to life and create discussion around how the Representative.VA.gov landing page will be similar to but have a distinct look and feel from the VA.gov landing page
- I am on the Accredited Representative Facing Team which owns the maintenance of this component
- The representatives app has flipper. Planned launch of the representatives app is June per [our roadmap](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1705012017021/1d76cf5fe7c9aab8ee495159f04a4e45853cdae8?sender=ub8a078a3d81c71eebb345899) so Flipper will likely be in place until then.

## Related issue(s)

- [#72960 Implement landing page first draft](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72960)

## Testing done

- Before this change, the user would navigate to /representatives and see the Representatives Dashboard
- To test these changes:
  - Navigate to /representatives to see the Landing Page
  - Click the `Until sign in is added use this to see dashboard` to link to the Dashboard with the url /representatives/dashboard
- Unit tests test existence of multiple key aspects of the Landing Page

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   No Page     |   <img width="304" alt="Screenshot 2024-01-24 at 10 35 29 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/6ac3c8e1-7387-45de-8e0a-1a779e03fd0f">    |
| Desktop |   No Page     |   <img width="1903" alt="Screenshot 2024-01-24 at 10 35 05 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/c43caf52-70ad-418f-8da1-37be324fbea2">    |

## What areas of the site does it impact?

- All changes are within src/applications/representatives. The primary change to this directory is that the child of the App is now the Landing Page (a new page styled to mirror the VA.gov landing page with a slight different look-and-feel styling) which has a Link to the Dashboard Page

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
